kubernetes/kubernetes.libsonnet

```diff
@@ -340,6 +341,14 @@ local letsbuildServiceDeployment(
       + hpa.spec.behavior.scaleUp.withStabilizationWindowSeconds(dc.autoscaling.behavior.scaleUp.stabilizationWindowSeconds)
   ),
 
+  pdp: (
+    if dc.autoscaling.enabled
+    then
+    pdp.new(dc.name)
+      + pdp.spec.withMaxUnavailable(dc.autoscaling.minReplicas - 1)
```

Comment:
there are actually a few repos that have minReplicas set to 1. Mostly some geniebelt ones. Either we need to force minReplicas or we need to force this is never less than 1

Reply:
Fixed